### PR TITLE
Add app veyor

### PR DESCRIPTION
**Pull request by erwan-joly**
notifyreviewer_-_appveyor

### CODE REVIEW

File: deploy.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const zipFolder = require('zip-folder');
+const editJsonFile = require("edit-json-file");
+const fs = require('fs');
+
+let file = editJsonFile('./dist/manifest.json');
```
Comment (chris-findlay) [inline deploy.js]: Why `let` and not `const`?  Here and elsewhere.

File: deploy.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }).catch((error) => {
+      console.log(`Error while publishing uploaded extension: ${error}`);
+      process.exit(1);
+    });
+
+  }).catch((error) => {
+    console.log(`Error while uploading ZIP: ${error}`);
+    process.exit(1);
+  });
+}
```
Comment (chris-findlay) [inline deploy.js]: no trailing newline

**Comment by chris-findlay** [comment]
I don't know anything about `appveyor` so am not sure I would spot any errors, but what I do recognise looks good, apart from the minor things above.

File: deploy.js (hunk 1 above)
Comment (chris-findlay) [reply deploy.js]: Also below.  I didn't see any `let` that needed to be there so the var could be mutated later...

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+environment:
+  nodejs_version: "6"
+  EXTENSION_ID:
+    secure: 1sDEVQM+HCPKqyAwafg5BUi7d+fVcyoYuzxuBRGfsRyju2QC7YVy9ha9Ze6aNZMZ
```
Comment (gregsdennis) [inline appveyor.yml]: Probably don't want these in code.  In you project in Appveyor, go to the settings (⚙️ to the right), then Environment, and you can add them there.

Comment (erwan-joly) [reply appveyor.yml]: this is a secured version i need them as env variable in the npm. secure value should be decrypted only for pushpay-labs org member pr (following the configuration on screenshot)

Comment (leafgarland) [reply appveyor.yml]: I don't know which is better (env vars or secure vars) - but Erwan's usage of secure variables seems right here according to [appveyor docs](https://www.appveyor.com/docs/build-configuration/#secure-variables). These values are encrypted already.

Comment (gregsdennis) [reply appveyor.yml]: How does appveyor decrypt it? Do you store a key anywhere?

Comment (leafgarland) [reply appveyor.yml]: Appveyor generates a key for each account

**Pull request merged**